client/vscode/src/api.ts

```diff
@@ -10,15 +18,25 @@ export interface ExtensionApi {
      * If this API changes, the version number will be incremented.
      */
     apiVersion(version: 1): {
+        /**
+         * Get OpenCtx provider capabilities.
+         */
+        capabilities(params: CapabilitiesParams): Promise<CapabilitiesResult[] | null>
```

Comment:
@sqs we introduced this so that the vscode client can ask the provider what item types it provides. This is useful in the new autocomplete stuff around mentions where it lists all the types.

Reply:
makes sense